builder/FindPackages.cmake

```diff
@@ -494,7 +494,6 @@ if( Linux )
   pkg_check_modules( PKG_MFX       libmfx>=1.28 )
 
   if ( PKG_X11_FOUND AND PKG_LIBVA_X11_FOUND )
-    set( ENABLE_X11 true )
```

Comment:
Design from the perspective: if ENABLE_X11=ON and X11 packages were not found - error out.